include/boost/url/bnf/parse.hpp

```diff
@@ -77,214 +96,182 @@ parse(
     error_code& ec,
     char ch) noexcept;
 
-/** @brief Parse a sequence of elements
+/** Parse beginning of a char range as a sequence of arbitrary BNF elements
 
    This function parses the string defined by the first chars in the
-   range of chars [@ref it, @ref end) into the Backus–Naur form (BNF)
-   elements [arg_0, arg_1, ..., arg_n] in @ref args.
+   range of chars [`it`, `end`) into the Backus–Naur form (BNF)
+   elements [`arg_0`, `arg_1`, ..., `arg_n`] in `args`.<br>
 
-   For each type Arg_i in [Arg_0, Arg_1, ..., Arg_n] in @ref Args, a
-   corresponding function overload
-   `bool parse(char const*& it, char const* const end, error_code&, Arg_i const& t)`
-   should be defined to consume the input from [@ref it, @ref end) into `arg_i`.
+   For each type `Arg_i` in [`Arg_0`, `Arg_1`, ..., `Arg_n`] in `Args`, a
+   corresponding function overload of `parse`
+   `bool parse(char const*& it, char const* const end, error_code&, Arg_i const& arg_i)`
+   should be defined to consume the input from [`it`, `end`) into `arg_i`.<br>
 
-   This `parse` function is defined for `char`s so that it matches the BNF
-   string literals in [@ref it, @ref end).
-
-   The function returns `true` and resets @ref ec when the input is
+   The function returns `true` and resets `ec` when the input is
    parsed successfully. If the input cannot be parsed into the BNF elements,
-   the function returns `false` and sets the error number in @ref ec to the
+   the function returns `false` and sets the error number in `ec` to the
    appropriate error number defined in @ref boost::urls::error.
 
```

Comment:
You should assume that people reading the docs know how `error_code` works. You don't have to tell them that it is set to a number when an error happens, and cleared on success.

Reply:
Oh... right. I did fix that for the other functions but then missed this one.